pakyow-support/lib/pakyow/support/dir_walk.rb

```diff
@@ -0,0 +1,49 @@
+module Pakyow
+  module Support
+    module WalkDir
+      DOT = ".".freeze
+
+      refine Dir.singleton_class do
```

Comment:
Cool. Thanks for the explanation. Still haven't used refinements too often. Quite useful though.